### DISCUSSION TITLE
能帮忙看下这条log是什么意思吗

### DISCUSSION THREAD

**Discussion by keeno1982** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#137] [Q&A] [answered]
openwrt系统日志里面满屏都是这条日志，是哪里出错了吗
Wed Mar 30 14:00:00 2022 cron.err crond[21270]: USER root pid 30802 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:05:00 2022 cron.err crond[21270]: USER root pid 10129 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:10:00 2022 cron.err crond[21270]: USER root pid 24207 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:15:00 2022 cron.err crond[21270]: USER root pid 2269 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:20:00 2022 cron.err crond[21270]: USER root pid 16377 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:25:00 2022 cron.err crond[21270]: USER root pid 31469 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:30:00 2022 cron.err crond[21270]: USER root pid 12967 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:35:00 2022 cron.err crond[21270]: USER root pid 24595 cmd /usr/share/unblockneteasemusic/log_check.sh
Wed Mar 30 14:40:00 2022 cron.err crond[21270]: USER root pid 7306 cmd /usr/share/unblockneteasemusic/log_check.sh

**Comment by pan93412** [comment]
我們沒有 `log_check.sh` 檔案呀⋯⋯？

**Comment by 1715173329** [comment, marked as answer]
这是正常现象，忽略即可。

**Reply by pan93412** [reply]
是不是該轉移到 install_scripts repo
